build/circle-deps.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -ex
+
+if [ "$1" = "docker" ]; then
+    cmds=$(grep '^cmd' GLOCKFILE | grep -v glock | awk '{print $2}')
+
+    # Restore previously cached build artifacts.
+    time go install github.com/cockroachdb/build-cache
+    time build-cache restore . .:race,test ${cmds}
+    # Build everything needed by circle-test.sh.
+    time go test -race -v -i ./...
+    time go install -v ${cmds}
+    time make install
+    time (cd acceptance; go test -v -c -tags acceptance)
+    # Cache the current build artifacts for future builds.
+    time build-cache save . .:race,test ${cmds}
+
+    exit 0
+fi
+
+gopath0="${GOPATH%%:*}"
+cachedir="${gopath0}/pkg/cache"
+tag="20150719-133944"
```

Comment:
The `cockroach/builder` image is versioned using this tag. If we add something new to the image, you run `build/builder.sh push` and then update the tag here. I'll add a comment explaining this.